opennms-base-assembly/src/main/filtered/etc/poller-configuration.xml

```diff
@@ -338,6 +338,27 @@
       <downtime begin="0" end="432000000" interval="300000"/><!-- 5m, 0, 5d -->
       <downtime begin="432000000" interval="3600000"/><!-- 1h, 5d -->
    </package>
+
+   <package name="device-config">
+      <filter>IPADDR != '0.0.0.0'</filter>
+      <rrd step="300">
+         <rra>RRA:AVERAGE:0.5:1:2016</rra>
+         <rra>RRA:AVERAGE:0.5:12:1488</rra>
+         <rra>RRA:AVERAGE:0.5:288:366</rra>
+         <rra>RRA:MAX:0.5:288:366</rra>
+         <rra>RRA:MIN:0.5:288:366</rra>
+      </rrd>
+      <service name="DeviceConfig-Default" interval="30000" user-defined="false" status="on">
+         <parameter key="retry" value="3"/>
+         <parameter key="timeout" value="3000"/>
+         <parameter key="rrd-repository" value="${install.share.dir}/rrd/response"/>
+         <parameter key="rrd-base-name" value="device-config"/>
+         <!-- match service name suffix with config-type -->
+         <parameter key="config-type" value="${requisition:config-type|Default}"/>
```

Comment:
I would like to see the other configuration parameter here, namely: `script`, `username`, `password`, `port`, and `timeout` (cf. `DeviceConfigMonitor`).